torchtext/experimental/datasets/raw/translation.py

```diff
@@ -0,0 +1,391 @@
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/test_2016.3.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/test_2016.3.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/test_2016.4.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/test_2016.4.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/test_2016.5.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/test_2016.5.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.1.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.1.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.2.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.2.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.3.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.3.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.4.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.4.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.5.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/train.5.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.1.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.1.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.2.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.2.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.3.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.3.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.4.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.4.en.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.5.de.gz",
+        "https://raw.githubusercontent.com/multi30k/dataset/master/data/task2/raw/val.5.en.gz"
+    ],
+    'WMT14':
+    'https://drive.google.com/uc?export=download&id=0B_bZck-ksdkpM25jRUN2X2UxMm8',
+    'IWSLT':
+    'https://wit3.fbk.eu/archive/2016-01//texts/{}/{}/{}.tgz'
+}
+
+
+def _read_text_iterator(path):
+    with io.open(path, encoding="utf8") as f:
+        reader = unicode_csv_reader(f)
+        for row in reader:
+            yield " ".join(row)
+
+
+def _clean_xml_file(f_xml):
```

Comment:
what's this func for?

Reply:
Some files in IWSLT are original in xml files, so need to have this function to extract the contents. This function was also already there from the branch you created previously, and it was also there in the old translation dataset class.